tests/unit/amo/components/TestAddonReviewCard.js

```diff
@@ -985,23 +994,29 @@ describe(__filename, () => {
     });
 
     it('renders a short byLine explicitly', () => {
-      const i18n = fakeI18n();
-      const review = signInAndDispatchSavedReview();
-      const root = render({ i18n, shortByLine: true, review });
+      const review = _setReview(fakeReview);
+      const root = render({ shortByLine: true, review });
 
       expect(
         renderByLine(root).prop('dangerouslySetInnerHTML').__html,
       ).toContain('posted ');
     });
 
     it('does not render a byLine for ratings', () => {
-      const review = signInAndDispatchSavedReview({
-        externalReview: fakeRatingOnly,
-      });
+      const review = _setReview(fakeRatingOnly);
       const root = render({ review });
 
       expect(root.find(UserReview)).toHaveProp('byLine', false);
     });
+
+    it('renders a short byLine for user profile reviews', () => {
+      const review = _setReview(fakeReview);
+      const root = render({ isUserProfile: true, review });
+
+      expect(
+        renderByLine(root).prop('dangerouslySetInnerHTML').__html,
```

Comment:
We usually use `.html()` instead of accessing this prop directly.

On the other hand, I see this way of accessing `dangerouslySetInnerHTML` a lot in this file..

Reply:
> On the other hand, I see this way of accessing `dangerouslySetInnerHTML` a lot in this file..

True, but I added all of them, so it's not like I was following a pre-existing pattern. I'll try `.html()` instead.